dev-scripts/prepare_release.sh

```diff
@@ -11,7 +11,17 @@ EMAIL="${opts[--email]}"
 VERSION="${opts[--version]}"
 NOTES="${opts[--notes]}"
 
-./prepare_package_xml.sh --name "${NAME}" --user "${USER}" --email "${EMAIL}" --version "${VERSION}" --notes "${NOTES}" >package.xml
+
+BASENAME=${0}
+TOOLS_DIR=$(dirname ${BASENAME})
+WORK_DIR="${TOOLS_DIR}/.."
+
+cd "${WORK_DIR}"
+
+git submodule update --init --recursive
+
+$TOOLS_DIR/cleanup.sh
+$TOOLS_DIR/prepare_package_xml.sh --name "${NAME}" --user "${USER}" --email "${EMAIL}" --version "${VERSION}" --notes "${NOTES}" >package.xml
 if [[ $? -ne 0 ]]; then
   echo "ERROR PROCESSING review package.xml"
   exit 1
```

Comment:
two things:
1) why doesn't this file just use `set -e` at the top so that it automatically fails? Manually checking seems like a recipe for disaster (what if the `git submodule update` call above fails? We forgot to check!)
2) I'd prefer we write our helper scripts be in python. I find shell scripts hard to use personally, and if they're in Python we can run them on Windows too. The longer we wait and bulk up our existing shell scripts, the more likely it is we'll just keep writing more shell scripts because that's just how it is in this repo

Reply:
I was just moving the helper scripts to a folder and didn't change much.

But, yeah, it will be better to rewrite them into a python script, will do.